templates/zerver/emails/onboarding_team_to_zulip.html

```diff
@@ -0,0 +1,40 @@
+{% extends "zerver/emails/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_uri }}/email_logo.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+
+<p>
+    {% trans %}If you've already decided to use Zulip for your organization, welcome! You can use our <a href="{{ get_organization_started }}">guide for setting up your organization</a> to get started.{% endtrans %}
+</p>
+<p>
+    {% trans %}Otherwise, here is some advice we often hear from customers for evaluating <i>any</i> team chat product:{% endtrans %}
+    <ol>
+        <li>{% trans %}<a href="{{ invite_users }}"><b>Invite your teammates</b></a> to explore with you and share their unique perspectives.{% endtrans %}
+            {% trans %}Use the app itself to chat about your impressions.{% endtrans %}
+        </li>
+        <li>{% trans %}<a href="{{ trying_out_zulip}}"><b>Run a week-long trial</b></a> with your team, without using any other chat tools. This is the only way to truly experience how a new chat app will help your team communicate.{% endtrans %}
+        </li>
+    </ol>
+</p>
+<p>
+    {% trans %}We hope these tips help you find a tool that <a href="{{ why_zulip }}">enables efficient communication</a> for your team.{% endtrans %}
```

Comment:
@alya one concern is that I would expect this link to take you to a blog post that's defining efficient conversation or explaining why it's important, which one might imagine would be pretty vague and not at all what we have behind that link.

It might be worth moving the "why zulip" link to a separate sentence for clarity that it's content specific to Zulip.

Reply:
Or revise it to be a stronger sell?

> Zulip is designed to [enable efficient communication](https://zulip.com/why-zulip/) for teams, so we hope you'll use the tips above to give it a try.

Reply:
Sure, we can try being less coy about it. Maybe something like this?

> Zulip is designed to [enable efficient communication](https://zulip.com/why-zulip/), and we hope these tips help your team experience it in action.